js/main.js

```diff
@@ -26,7 +34,118 @@ function saveFormData(event) {
   // reset form and image
   $form.reset();
   updatePhotoPreview(null);
+  // show entries
+  hideAllBut($views, 'entries');
 }
 
 $photoURL.addEventListener('input', updatePhotoPreview);
 $form.addEventListener('submit', saveFormData);
+
+// Render Entries (functions and event listeners)
+
+// Render journal entries according to sample HTML, get data from entryObj
+function renderEntry(entryObj) {
+  // entryObj will be an item in array data.entries
+  /* -- Sample Entry HTML --
+  <li data-entry-id=[entryObj.entryId]>
+    <div class="row">
+      <div class="column-half flex-col justify-center">
+        <img src=[entryObj.photoURL] alt="User selected image">
+      </div>
+      <div class="column-half">
+        <h2>[entryObj.title]</h2>
+        <p class="font-regular">
+          [entryObj.notes]
+        </p>
+      </div>
+    </div>
+  </li>
+  */
+
+  var $entryLi = document.createElement('li');
+  $entryLi.setAttribute('data-entry-id', entryObj.entryId);
+
+  var $entryRow = document.createElement('div');
+  $entryRow.className = 'row';
+
+  var $imgCol = document.createElement('div');
+  $imgCol.className = 'column-half flex-col justify-center';
+  var $txtCol = document.createElement('div');
+  $txtCol.className = 'column-half';
+
+  var $entryPhoto = document.createElement('img');
+  $entryPhoto.setAttribute('src', entryObj.photoURL);
+  $entryPhoto.setAttribute('alt', 'User selected image');
+
+  var $entryTitle = document.createElement('h2');
+  $entryTitle.textContent = entryObj.title;
+  var $entryNotes = document.createElement('p');
+  $entryNotes.textContent = entryObj.notes;
+  $entryNotes.className = 'font-regular';
+
+  $txtCol.appendChild($entryTitle);
+  $txtCol.appendChild($entryNotes);
+  $imgCol.appendChild($entryPhoto);
+
+  $entryRow.appendChild($imgCol);
+  $entryRow.appendChild($txtCol);
+
+  $entryLi.appendChild($entryRow);
+
+  return $entryLi;
+}
+
+/*
+hideAllBut - general function to hide views except desired one
+
+Provided a array of nodes of the data views and a string with the data view to show
+Loop through array
+  Check if the data-view attribute matches the desired one
+    If not, make it hidden by adjusting the class name
+  If it is, set the class name to an empty string
+If dataViewToShow === 'entries', we will want to render the entries tree again
+  Query for the ul element containing all of the entry list items (id entries-list)
+  Clear out all children nodes
+  If data.entries is empty
+    Create a new p element object and set its textContent to "No entries have been recorded"
+    Apply styling classes
+    Append as child to the entries list
+  Loop through data.entries and call renderEntry() on each
+  Append the created DOM tree for each entry to the now cleared entries list
+Update the data object with the now current view
+(This function should be called at the end of the 'submit' event listener)
+*/
+function hideAllBut(views, dataViewToShow) {
```

Comment:
I would rename this to something like `switchViews` to better illustrate its purpose.